scripts/gitlab/gitlab-ci.yml

```diff
@@ -0,0 +1,31 @@
+stages:
+  - build
+  - test
+  - deploy
+
+variables:
+  APP_NAME: Project
+
+before_script:
+  - echo "hello world"
+
+build:
+  stage: build
+  script:
+    - composer install
+    - blt setup:build
+
+validate:
+  stage: test
+  script:
+    - blt validate
+
```

Comment:
In pipelines you define various build stages and then tasks for each stage. This particular example was to demonstrate that within the `test` stage you can have different task executions, e.g. in this one only linting happens.